modules/yieldliftBidAdapter.js

```diff
@@ -0,0 +1,144 @@
+import {registerBidder} from '../src/adapters/bidderFactory.js';
+import * as utils from '../src/utils.js';
+import {BANNER} from '../src/mediaTypes.js';
+
+const ENDPOINT_URL = 'https://x.yieldlift.com/auction';
+
+const DEFAULT_BID_TTL = 30;
+const DEFAULT_CURRENCY = 'USD';
+const DEFAULT_NET_REVENUE = true;
+
+export const spec = {
+  code: 'yieldlift',
```

Comment:
I see that `yieldlift` is an alias for `bidphysics` adapter, should it stay that way, now that you created a dedicated bidAdapter for `yieldlift`?

Reply:
ok, cool, you should create separate MR for that, thanks for explaining.

Reply:
Problem is that due to some organizational shifts bidphysics domain is now deprecated, and email account we used for github is closed. So I cant access it and create new PR to remove bidphysics adapter...

Do you have any suggestion how to resolve this, in order to keep things clean? 

Thanks!